config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
-   get "/", to: "welcome#index"
-
+    get "/", to: "welcome#index"
+    get "/login", to: "sessions#new"
+    post "/login", to: "sessions#create"
+    get "/logout", to: "sessions#index"
```

Comment:
I really don't like doing it this way, I want to try to make a delete request to sessions#destroy, but need to figure out how to use button_to as it does not pick up the login path unless I do it with an <a> tag

Reply:
I think this is important to figure out before we merge this branch. I suggest just using links instead of buttons if the buttons are the problem. At this point function and following ReSTful conventions is more of a priority than using buttons. If anyone else disagrees, no problemo!

Reply:
changed it to link_to with delete method, the button still looks the same